### ISSUE TITLE
Writify bug

### ISSUE THREAD

**Issue by Supertubbis** [CharlieS1103/spicetify-extensions#71]
## Describe bug
when i add a note to a song the only way to view the note is by right clicking and clicking add note. it doesn't show up next to the song

## Steps to replicate
add writify to spicetify. right click on a song in a playlist. note does not appear.

## Necessary screenshots
https://imgur.com/a/zVQNTQY
https://imgur.com/a/b1hjXI9

## Spicetify and Spotify Version
0.8.9

please email me at packpackgamer@gmail.com if you need more info.

**Comment by CharlieS1103** [comment]
Yeah I still need to fix that but I haven't really been coding much over the summer

**Comment by Supertubbis** [comment]
> Yeah I still need to fix that but I haven't really been coding much over the summer

there's no rush it's just this is a thing i've wanted in spotify for a long while and was bummed that it was bugged

**Comment by CharlieS1103** [comment]
Yup, sorry about that. Issues is that when I coded this extension I coded it to work with one specific theme (Comfy's) grid layout and it didn't work for any other themes

**Comment by Supertubbis** [comment]
> Yup, sorry about that. Issues is that when I coded this extension I coded it to work with one specific theme (Comfy's) grid layout and it didn't work for any other themes

are you planning on patching this soon?

**Comment by Brenzle** [comment]
> Yup, sorry about that. Issues is that when I coded this extension I coded it to work with one specific theme (Comfy's) grid layout and it didn't work for any other themes

I installed Writify and downloaded that theme but the notes column was not showing. I then moved writify.js to Local/spicetify/Extensions and changed the adress to the globals.d.ts file since it was looking for it in the Roaming/spicetify-cli but my file was in Local/spicetify. I also uncommented the TODO part from line 25 and got the notes column to show up but empty, without any notes. My notes do get saved but not displayed in the column. Is there something I can do to get it to work for the Comfy theme again?
